### PR TITLE
Feature/invoice

### PR DESCRIPTION
**Pull request by g1st**
### Proposal
Add invoices.

### Checklist

- [x] I have made this pull request to the `master` branch
- [x] I have run `npm run lint` and there are no errors

### CODE REVIEW

File: client/src/components/Reports/Invoice/InvoiceByCustomer.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+					key={i}
+					data={customerAddresses}
+					addressTotals={data.addressTotals}
+				/>
+			))}
+			<tbody>
+				<TotalsRow data={data.generalTotals} />
+			</tbody>
+		</Table>
+	);
```
Comment (GTabala) [inline client/src/components/Reports/Invoice/InvoiceByCustomer.js:38]: I miss the BackButton component.

File: client/src/components/Reports/Invoice/InvoiceByCustomerShort.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+					key={i}
+					data={customerAddresses}
+					addressTotals={data.addressTotals}
+				/>
+			))}
+			<tbody>
+				<TotalsRowShort data={data.generalTotals} />
+			</tbody>
+		</Table>
+	);
```
Comment (GTabala) [inline client/src/components/Reports/Invoice/InvoiceByCustomerShort.js:33]: I miss the BackButton component.

**Review by GTabala — CHANGES_REQUESTED** [review]

File: client/src/components/Reports/Invoice/InvoiceByCustomer.js (hunk 1 above)
Comment (GTabala) [reply client/src/components/Reports/Invoice/InvoiceByCustomer.js:38]: Also, it is good to provide an opportunity to open a certain job for editing.

File: client/src/components/Reports/Invoice/InvoiceByCustomerShort.js (hunk 2 above)
Comment (GTabala) [reply client/src/components/Reports/Invoice/InvoiceByCustomerShort.js:33]: Also, it is good to provide an opportunity to open a certain job for editing.

File: client/src/components/Reports/Invoice/InvoiceByCustomer.js (hunk 1 above)
Comment (g1st) [reply client/src/components/Reports/Invoice/InvoiceByCustomer.js:38]: Oops, thanks.

**Review by GTabala — APPROVED** [review]
Good!

**Pull request merged**